js/index.js

```diff
@@ -1 +1,6 @@
 export default function igArticle(){};
+
+igArticle.init = function() {
+  window.isNext = document.cookie.replace(/(?:(?:^|.*;\s*)FT_SITE\s*\=\s*([^;]*).*$)|^.*$/, "$1");
```

Comment:
Putting this here means the classes will be added to the page too late and will result in a FOUC. I'm not sure what alternative to recommend.